packages/qwik-city/runtime/src/types.ts

```diff
@@ -364,13 +364,15 @@ export type FailOfRest<REST extends readonly DataValidator[]> = REST extends rea
 /** @public */
 export type ValidatorErrorKeyDotNotation<T, Prefix extends string = ''> = T extends object
   ? {
-      [K in keyof T & string]: T[K] extends (infer U)[]
-        ? U extends object
-          ? `${Prefix}${K}[]` | `${Prefix}${K}[]${ValidatorErrorKeyDotNotation<U, '.'>}`
-          : `${Prefix}${K}[]`
-        : T[K] extends object
-          ? ValidatorErrorKeyDotNotation<T[K], `${Prefix}${K}.`>
-          : `${Prefix}${K}`;
+      [K in keyof T & string]: {} extends T[K]
```

Comment:
Can you test this type? You can find examples of type tests in the `packages/qwik/src/core/render/jsx/types/jsx-types.unit.tsx` for example.

Reply:
I will check it out after my short vacation on Thursday.

I think this change will remove the dot notation, but let me test everything. 

The idea of testing the type is perfect, will do it too.